packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTableRowAction.tsx

```diff
@@ -33,11 +35,17 @@ export function TransactionsTableRowAction({
   isError: boolean
   type: 'deposits' | 'withdrawals'
 }) {
+  const { address: connectedAddress } = useAccount()
   const { chain } = useNetwork()
   const { switchNetworkAsync } = useSwitchNetworkWithConfig()
   const networkName = getNetworkName(chain?.id ?? 0)
   const { sanitizedAddress } = useTransactionHistoryAddressStore()
 
+  const viewingAnotherAddress =
+    connectedAddress &&
+    sanitizedAddress &&
```

Comment:
NIT: Wouldn't it be better to rename the addresses to `searchedAddress` and `connectedAddress`, since that's more apt?